### ISSUE TITLE
Debug log calls left in code. Puking garbage into the console.

### ISSUE THREAD

**Issue by alpalmerxactware** [sparksuite/simplemde-markdown-editor#316]
A "console.log(x);" and "console.log(items[x]);" have been left in the "// Don't include trailing separators" loop. I believe these were used for debugging but are currently in the distributed version and making a mess of the browser console.

**Comment by everthis** [comment]
+1

**Comment by anhtran** [comment]
Checkout #280

**Comment by alpalmerxactware** [comment]
Maybe I missed something. But #280 was closed over a month ago with u/WesCossick making a reference to #177 which he then left open but placed a circular reference BACK to #280
Returning to #280 we see a reference to #284 in which these problematic console.log calls have been removed via a merge request posted by u/foreachlt. Said code was merged into the development branch But has NOT made it into the master branch. Which is a pretty sizable problem for everyone who gets this library via Bower. So is there some way we can expedite the changes in question being merged to Master?

**Comment by WesCossick** [comment]
#177 is a verbose logging option feature. It is different than accidental `console.log()` code leftover in the minified file.

The development branch is merged with the master branch once the milestone is complete. You can see the progress here: https://github.com/NextStepWebs/simplemde-markdown-editor/milestones/1.11.0. If you'd like to expedite it, please submit PRs for the issues included in that milestone.
